test/MusicStore.E2ETests/Implementation/OpenIdConnectLoginScenarios.cs

```diff
@@ -98,12 +107,21 @@ public partial class Validator
             };
 
             content = new FormUrlEncodedContent(formParameters.ToArray());
+
             // Need a non-redirecting handler
             var handler = new HttpClientHandler() { AllowAutoRedirect = false };
             handler.CookieContainer.Add(new Uri(_deploymentResult.ApplicationBaseUri), _httpClientHandler.CookieContainer.GetCookies(new Uri(_deploymentResult.ApplicationBaseUri)));
             _httpClient = new HttpClient(handler) { BaseAddress = new Uri(_deploymentResult.ApplicationBaseUri) };
 
             response = await DoPostAsync("Account/LogOff", content);
+            Assert.True(response.Headers.TryGetValues("Set-Cookie", out var logoutSetCookieHeaders));
+            var responseCookies = SetCookieHeaderValue.ParseList(logoutSetCookieHeaders.ToList());
+
+            foreach (var logoutSetcookie in logoutSetCookieHeaders)
+            {
+                _httpClientHandler.CookieContainer.SetCookies(new Uri(_deploymentResult.ApplicationBaseUri), logoutSetcookie.Replace("; samesite=lax", ""));
```

Comment:
I'm just trying to fix the test. I leave any bigger change to the discretion of @kichalla. Flowing the cookies manually is not a trivial process and is error prone.